### ISSUE TITLE
Volume control on playback device without function

### ISSUE THREAD

**Issue by MrShoenel** [duncanthrax/scream#4]
When the Scream WDDM device is selected as output device, one cannot control the volume/loudness. More specifically, it is only possible to switch the sound on or off. If the fader is set to volume = 0, the sound is off; any other volume setting will only turn the sound on (i.e. there is no volume level adjustment possible).

This was tested on Windows 10 (1703) x64 with the latest signed drivers.

**Comment by Delekhan** [comment]
I can confirm this problem with the latest release. I investigated a little in die Windows Driver Docs and source and may found the cause of the missing volume control.

As per the Microsoft doc https://docs.microsoft.com/en-us/windows-hardware/drivers/audio/software-volume-control-support the property KSPROPERTY_AUDIO_VOLUMELEVEL indicated weather the driver has hardware volume control support or not. In the scream source in toptable.h this property is set and thus maybe blocking volume control.

Is it possible to build the driver without KSPROPERTY_AUDIO_VOLUMELEVEL and test wether this helps? I failed in building and signing the drivers by myself :-(

Thanks!
